src/dex/uniswap-v2/uniswap-v2.ts

```diff
@@ -45,11 +45,11 @@ import { Contract } from 'web3-eth-contract';
 import { UniswapV2Config, Adapters } from './config';
 import { BI_MAX_UINT } from '../../bigint-constants';
 
-const RESERVE_LIMIT = 2n ** 112n - 1n;
+export const RESERVE_LIMIT = 2n ** 112n - 1n;
```

Comment:
Importing order do not give me move `RESERVE_LIMIT` to `src/dex/uniswap-v2/constants.ts`: when Ido that, then I get error in `src/dex/uniswap-v2/mdex.ts:22:33`: 

> TypeError: Class extends value undefined is not a constructor or null

Because `constants.ts` imports few uniswap fork dex configs what extends UniswapV2 class that is not yet compiled..

Reply:
I see. Thanks for pointing it out. Then let's leave it as it is